scripts/hal_alignment/hal_gene_liftover.py

```diff
@@ -31,12 +31,13 @@
 """
 
 from argparse import ArgumentParser
+import io
```

Comment:
```suggestion
```

Remove import if you follow suggestion below which removes call to `io.StringIO`